biogrid/libs/biogrid-simulator/src/lib/config/bio-constants.ts

```diff
@@ -136,3 +136,20 @@ export interface ShortestDistances {
 export const TIME = {
   DISCRETE_UNIT_HOURS: 1,
 };
+
+export class MAIN_GRID implements GridItem {
+  gridItemName: string = GRID_ITEM_NAMES.GRID;
+  gridItemResistance: number = RESISTANCE.GRID;
+  // 
```

Comment:
delete?

Reply:
yeah, I removed it. This was in the previous commit